docs/install-kafka-operator.md

```diff
@@ -35,74 +36,264 @@ This method uses a command-line tool of the commercial [Banzai Cloud Supertubes]
     supertubes install -a
     ```
 
-## Install {{< kafka-operator >}} and the requirements independently {#manual-install}
+## Install {{< kafka-operator >}} and its requirements independently {#install-kafka-operator-and-its-requirements-independently}
 
-### Install cert-manager {#install-cert-manager}
+### Install cert-manager with Helm {#install-cert-manager-with-helm}
 
-{{< kafka-operator >}} uses [cert-manager](https://cert-manager.io) for issuing certificates to clients and brokers. Deploy and configure cert-manager if you haven't already done so.
+{{< kafka-operator >}} uses [cert-manager](https://cert-manager.io) for issuing certificates to clients and brokers and cert-manager is required for TLS-encrypted client connections. It is recommended to deploy and configure a cert-manager instance if there is none in your environment yet.
 
 > Note:
-> - {{< kafka-operator >}} 0.18.1 and newer supports cert-manager 1.5.3-1.6.x
+> - {{< kafka-operator >}} 0.24.0 and newer versions support cert-manager 1.10.0+ (which is a requirement for Red Hat OpenShift)
+> - {{< kafka-operator >}} 0.18.1 and newer supports cert-manager 1.5.3-1.9.x
 > - {{< kafka-operator >}} 0.8.x-0.17.0 supports cert-manager 1.3.x
 
-Install cert-manager and the CustomResourceDefinitions using one of the following methods:
+1. Install cert-manager's CustomResourceDefinitions.
+
+    ```bash
+    kubectl apply \
+    --validate=false \
+    -f https://github.com/jetstack/cert-manager/releases/download/v1.11.0/cert-manager.crds.yaml
+    ```
+
+    Expected output:
+
+    ```bash
+    customresourcedefinition.apiextensions.k8s.io/certificaterequests.cert-manager.io created
+    customresourcedefinition.apiextensions.k8s.io/certificates.cert-manager.io created
+    customresourcedefinition.apiextensions.k8s.io/challenges.acme.cert-manager.io created
+    customresourcedefinition.apiextensions.k8s.io/clusterissuers.cert-manager.io created
+    customresourcedefinition.apiextensions.k8s.io/issuers.cert-manager.io created
+    customresourcedefinition.apiextensions.k8s.io/orders.acme.cert-manager.io created
+    ```
+
+1. If you are installing cert-manager on a Red Hat OpenShift version **4.10** cluster, the default security computing profile must be enabled for cert-manager to work.
```

Comment:
Good point, I will take that out.

Reply:
I mean, let's finish the envoy 4.10 test and then Koperator can claim 4.10 support because we tested that.
Doesn't matter that others building on top of Koperator will clear 4.10 or not.